group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java

```diff
@@ -5567,7 +5600,7 @@ public void testJoinGroupAppendErrorConversion() {
         assertEquals(Errors.LEADER_NOT_AVAILABLE, Errors.LEADER_NOT_AVAILABLE);
     }
 
-    private <T> void assertUnorderedListEquals(
+    public static <T> void assertUnorderedListEquals(
```

Comment:
Is there a reason for making it public? I have the same question for the following two methods.

Reply:
My bad I had to only make the recordEquals method public to be able to use in the RecordHelpersTest, reverted the others to private

Reply:
I see. I wonder if we should actually move `recordEquals` to `RecordHelpersTest`. The dependency would make more sense this way, no?

Reply:
Yeah that makes sense, I'll have to move all the corresponding methods also to that class